service/src/main/java/com/shortlink/service/server/Base62LtS.java

```diff
@@ -15,9 +15,9 @@ private String base62(long num) {
     }
 
     public String encode(String url){
-        long numUrl = Long.parseLong(url);
+        long numUrl = url.split("https://www.")[1].split(".com")[0].hashCode();
```

Comment:
Why split instead of using the whole URL?

Reply:
I thought the hashcode() function cannot take special characters like "://", but it turns out that it can take these chars. I have changed it.